libraries/package_deps.rb

```diff
@@ -6,7 +6,9 @@ def install_ruby_dependencies
         when /^jruby-/
           package jruby_package_deps
         else
-          package package_deps
+          package_deps.each do |deps|
+            package deps
+          end
```

Comment:
this line shouldn't be needed. 

It was probably erroring out regardless of this line due to line 39 missing.

Reply:
I did that because, after put dependecies for `mac_os_x`, I got this error:
```
      ================================================================================
      Error executing action `install` on resource 'homebrew_package[openssl, makedepend, pkg-config, libyaml, libffi]'
      ================================================================================

      Chef::Exceptions::Package
      -------------------------
      No candidate version available for makedepend, pkg-config, libyaml, libffi

      Resource Declaration:
      ---------------------
      # In /var/chef/cache/cookbooks/ruby_rbenv/libraries/package_deps.rb

        9:           package package_deps
       10:         end

      Compiled Resource:
      ------------------
      # Declared in /var/chef/cache/cookbooks/ruby_rbenv/libraries/package_deps.rb:9:in `install_ruby_dependencies'

      homebrew_package("openssl, makedepend, pkg-config, libyaml, libffi") do
        package_name ["openssl", "makedepend", "pkg-config", "libyaml", "libffi"]
        action [:install]
        default_guard_interpreter :default
        declared_type :package
        cookbook_name "gbm-rbenv"
      end
```
And
```
Chef::Exceptions::Package
    -------------------------
    homebrew_package[openssl, makedepend, pkg-config, libyaml, libffi] (/var/chef/cache/cookbooks/ruby_rbenv/libraries/package_deps.rb line 9) had an error: Chef::Exceptions::Package: No candidate version available for makedepend, pkg-config, libyaml, libffi
```

Reply:
Any suggestions?

Reply:
ah! homebrew doesn't accept multi package installs on the version you're running on. Which Chef-version is that?

Reply:
Chef: 13.6.4

Reply:
Ah right, can you try this on 14.latest for me please?

Reply:
I mean, we still support 13.x until next April so this will have to do.

Reply:
> Ah right, can you try this on 14.latest for me please?

Yes, I'll do